### ISSUE TITLE
Console error while opening child reports

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#24349]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Click on FAB -> Assign Task
2. Go to task report
3. Go to parent Report
4. Go to same task task from parent report
5. Observe Console error.
You can open any IOU report to reproduce this error

## Expected Result:
There should be no console error

## Actual Result:
There's a console error

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [x] Android / Chrome
- [x] iOS / native
- [x] iOS / Safari
- [x] MacOS / Chrome / Safari
- [x] MacOS / Desktop

**Version Number:** 1.3.49-1
**Reproducible in staging?:** Y 
**Reproducible in production?:** Y 
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 

https://github.com/Expensify/App/assets/43995119/0d191875-488d-46a7-8a59-314575cbe61f

**Expensify/Expensify Issue URL:**
**Issue reported by:** @nodebrute
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1691035111039979

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by jfquevedol2198** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Console error while opening child reports

### What is the root cause of that problem?
`parentReportID`, which is one of props of `ParentNavigationSubtitle` component is defined as `string`, but `number` type is passed.

### What changes do you think we should make in order to solve the problem?
https://github.com/Expensify/App/blob/58dc0c9576f8c3d3fa8924a7fec0889232c4b537/src/components/ParentNavigationSubtitle.js#L21

Update `parentReportID` type like this:
```javascript
parentReportID: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
```

### What alternative solutions did you explore? (Optional)
N/A

**Comment by cristipaval** [comment]
yes this should be held on https://github.com/Expensify/Expensify/issues/306294

**Comment by anmurali** [comment]
Handled internally and held on https://github.com/Expensify/Expensify/issues/306294

**Comment by anmurali** [comment]
Held.

**Comment by anmurali** [comment]
Still held but I see the issue is being actively worked on.

**Comment by anmurali** [comment]
Asked in the issue this is held on what our ETA is as the PR appears to have slowed down.

**Comment by anmurali** [comment]
Still held on https://github.com/Expensify/Expensify/issues/306294
Bumped that issue again. Going to change the prioritization of this one to a Monthly since it looks like the held PR is going slowly.

**Comment by anmurali** [comment]
Making this a monthly as the issues are console issues not related to customer and so are lower priority

**Comment by anmurali** [comment]
Still held on https://github.com/Expensify/Expensify/issues/306294 - asked for an update

**Comment by anmurali** [comment]
@cristipaval - https://github.com/Expensify/Expensify/issues/306294 is now closed. Can we take this off hold?

**Comment by cristipaval** [comment]
@anmurali, it seems that this issue is not reproducible anymore.
